x/slashing/handler.go

```diff
@@ -2,8 +2,27 @@ package slashing
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/pkg/errors"
+	"strings"
 )
 
+const (
+	SideChainIdBsc  int64 = 56
+	SideChainIdTest int64 = 97
+)
+
+func SideChainIdFromText(str string) (int64, error) {
+	switch strings.ToLower(str) {
+	case "bsc":
+		return SideChainIdBsc, nil
+	case "test":
```

Comment:
The testnet of BSC is named `chapel`, please use `chapel` instead of `test`

Reply:
ok